check-ntpoffset/lib/check-ntpoffset.go

```diff
@@ -160,49 +161,94 @@ func getNTPOffsetFromNTPServers(ntpServers string) (offset float64, err error) {
 	}
 }
 
-func getNTPOffsetFromNTPD() (offset float64, err error) {
-	err = withCmd(exec.Command(cmdNTPq, "-c", "rv 0 offset"), func(out io.Reader) error {
-		offset, err = parseNTPOffsetFromNTPD(out)
+func getNTPOffsetFromNTPD(checkStratum bool) (offset float64, err error) {
+	err = withCmd(exec.Command(cmdNTPq, "-c", "rv 0 stratum,offset"), func(out io.Reader) error {
```

Comment:
Here `stratum,` can be omitted if checkStratum is false, right?

Reply:
Yes, but I think it does not cost much and always specifying `stratum` is simpler.